DepositeCalcTests/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,71 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class HistoryPage : BasePage, IPage
+    {
+        public HistoryPage(IWebDriver driver) : base(driver)
+        {
+        }
+
+        private IWebElement CalculatorLnk => driver.FindElement(By.XPath("//div[text() = 'Calculator']"));
+        private IWebElement ClearBtn => driver.FindElement(By.XPath("//button[text() = 'Clear']"));
+        private IWebElement HistoryTableRow1Coloumn1Field => driver.FindElement(By.XPath("//table/tr[1]/th[1]"));
+        private IWebElement HistoryTableRow1Coloumn2Field => driver.FindElement(By.XPath("//table/tr[1]/th[2]"));
+        private IWebElement HistoryTableRow1Coloumn3Field => driver.FindElement(By.XPath("//table/tr[1]/th[3]"));
+        private IWebElement HistoryTableRow1Coloumn4Field => driver.FindElement(By.XPath("//table/tr[1]/th[4]"));
+        private IWebElement HistoryTableRow1Coloumn7Field => driver.FindElement(By.XPath("//table/tr[1]/th[7]"));
+        private IWebElement HistoryTableRow1Coloumn8Field => driver.FindElement(By.XPath("//table/tr[1]/th[8]"));
+        private IList<IWebElement> HistoryTableRowsList => driver.FindElements(By.XPath("//tr[@class='data-td']"));
+        private IWebElement HistoryTableRow2Coloumn1Field => driver.FindElement(By.XPath("//table/tr[2]/td[1]"));
+        private IWebElement HistoryTableRow2Coloumn2Field => driver.FindElement(By.XPath("//table/tr[2]/td[2]"));
+        private IWebElement HistoryTableRow2Coloumn3Field => driver.FindElement(By.XPath("//table/tr[2]/td[3]"));
+        private IWebElement HistoryTableRow2Coloumn7Field => driver.FindElement(By.XPath("//table/tr[2]/td[7]"));
+        private IWebElement HistoryTableRow2Coloumn8Field => driver.FindElement(By.XPath("//table/tr[2]/td[8]"));
+
+        public bool IsOpened()
+        {
+            return driver.Url.Contains("History");
+        }
+
+        public CalculatorPage Calculator()
+        {
+            CalculatorLnk.Click();
+            return new CalculatorPage(driver);
+        }
+
+        public HistoryPage Clear()
+        {
+            ClearBtn.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => ClearBtn.Enabled);
+            return new HistoryPage(driver);
+        }
+
+        public string HistoryTableRow1Column1 => HistoryTableRow1Coloumn1Field.Text;
```

Comment:
Good work, but it's better to use lists or arrays. E.g. to create such properties:
```
List<string> Headers => ...
List<string> LastRow => ...
List<List<string>> TableValues => ...
```